src/client/app/data.json

```diff
@@ -196,6 +223,6 @@
             "ClothCode": "Pt502",
             "ClothImage": "app/tmp/icon/order-pant.jpg"
         }
-
-    }
+            }
+        
```

Comment:
please remove unnescarry spaces , and before pushing the code , do the indentation check for json